app/assets/stylesheets/pageflow/themes/legacy/overview.scss

```diff
@@ -0,0 +1,48 @@
+.overview {
+  @import "pageflow/themes/default/overview/chapter_border_images";
+
+  font-family: sourcesanspro;
+
+  &:after {
+    background-color: black;
+  }
+
+  &.active:after {
+    opacity: 0.4;
+  }
+
+  .content {
+    color: #fff;
+
+    h2 {
+      font-size: 3em;
+    }
+
+    .ov_chapter {
+      .heading-4 {
+        font-weight: bold;
+        font-size: 1.1em;
+      }
+
+      .heading-5 {
+        font-size: 1em;
+        text-transform: uppercase;
+        font-weight: normal;
+      }
+
+      .ov_page {
+        border: 1px solid #505050;
+        color: #fff;
+
+        &.active {
+          border-color: #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.